pkg/server/telemetry/telemetry.go

```diff
@@ -0,0 +1,24 @@
+package telemetry
+
+import (
+	"github.com/armon/go-metrics/datadog"
+)
+
+type MetricSink interface {
+	IncrCounter([]string, float32)
```

Comment:
can you name the variables and add some comment so it's clear what these values are supposed to be? I imagine `[]string` is for labels and `float32` is for actual time series value?